### PR TITLE
[IMP] docstrings `map_values`

### PR DESCRIPTION
**Pull request by blaggacao**
suggest usage in post-migration.
please merge, if true, please close, if not.

### CODE REVIEW

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -703,6 +703,8 @@ def map_values(
     """
     Map old values to new values within the same model or table. Old values
     presumably come from a legacy column.
+    Unless special cases, you want to typically use it
```
Comment (StefanRijnhart) [inline openupgradelib/openupgrade.py]: 'You will typically want to use it in post-migration scripts.'

**Comment by pedrobaeza** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
Thanks for the change. If you silently update the code to honour the comments, the reviewers are not notified so next time, please add a comment as well saying that you did so.
:+1:

**Comment by blaggacao** [comment]
OK, cool. I wasn't aware of that github's behaviour. Thanks and Best, David

**Pull request merged**